vertx-web/src/main/java/io/vertx/ext/web/impl/RouteImpl.java

```diff
@@ -377,8 +389,15 @@ private void setPath(String path) {
     }
   }
 
+  private static final Pattern RE_NAMED_CAPTURING_GROUP = Pattern.compile("\\(\\?<([a-zA-Z][a-zA-Z0-9]*)>.*?\\)");
```

Comment:
Group name pattern created accordings to https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#groupname